.github/workflows/ci.yml

```diff
@@ -56,11 +62,29 @@ jobs:
 
       - name: Install SATySFi
         run: |
-          opam install -y .
+          opam install .
           ./install-libs.sh ~/.satysfi
 
       - name: Build demo docs
         run: |
           (cd demo; opam exec -- make)
           (cd doc; opam exec -- make)
           (cd tests; opam exec -- make)
+
+  lint-opam:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout tree
+        uses: actions/checkout@v4
+
+      - name: Set-up OCaml
+        uses: ocaml/setup-ocaml@v3
+        with:
+          ocaml-compiler: "4.14"
```

Comment:
Should this be `"4.14"` instead of `${{ matrix.ocaml-compiler }}` for the purpose of linting? I'm not sure which is more appropriate 😅

Reply:
You don't need to have multiple compiler versions except for build and test, so you just need to specify the latest supported compiler version!

Reply:
Ok, thanks!